charts/loki-distributed/templates/ingester/statefulset-ingester.yaml

```diff
@@ -7,7 +7,9 @@ metadata:
     {{- include "loki.ingesterLabels" . | nindent 4 }}
     app.kubernetes.io/part-of: memberlist
 spec:
+{{- if not .Values.ingester.autoscaling.enabled }}
```

Comment:
for consistency with the querier templates (which can also be run as either a deployment or a stateful set), I would advocate we not make replicas conditional on autoscaling here.